modules/network-observability-includelist-example.adoc

```diff
@@ -0,0 +1,31 @@
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-netobserv-dashboard-high-traffic-alert_{context}"]
+= Creating an alert for high ingress traffic
```

Comment:
Sorry for the late suggestion, but when reading this we could think we can just create alerts for that very precise use case.

It could perhaps be reformulated as "Creating alerts" as a title, change the first paragraph sentence to something like:

> You can create custom rules for the *Netobserv* metrics to trigger alerts when some defined conditions are met.

And then finally introduce that special case on ingress traffic like you did.

wdyt?

Reply:
No worries, any and all feedback at any time is good :)  I can see what you mean here. This procedure could be misinterpreted as the _only_ for this use case. I take your suggestion

Reply:
Addressed. TY!